docs/content/getting-started/production-environment.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Running OpenFGA in Production
+description: Running OpenFGA in a Production Environment
+sidebar_position: 8
+slug: /getting-started/running-in-production
+---
+
+import {
+  DocumentationNotice,
+  RelatedSection,
+} from "@components/Docs";
+
+# Running OpenFGA in Production
+
+<DocumentationNotice />
+
+## Preparation
```

Comment:
I would remove this and just have a top-level paragraph that says something like

The following list outlines some guidelines and best-practices for running OpenFGA in a production environment:

* [Configure Authentication](./setup-openfga.mdx#configuring-authentication)
* Enable HTTP TLS or gRPC TLS or both
* Configure JSON as the `log.format`
* [Disable the Playground](./setup-openfga.mdx#playground)


Note: We need to add info regarding TLS for the HTTP and gRPC servers. We support TLS, but we don't have it documented in the Setup page yet. Ideally the TLS bullet point above would have links to those sections.